frontend/styles/var/mixins.scss

```
@@ -493,3 +493,36 @@ $max-width: 2560px;
+// Shared styling for a search input rendered inside a dropdown menu (icon on
+// the left, 36px tall, Fleet-black-10 border). Used by FleetsDropdown,
+// CategoryFilter, and ActivityTypeDropdown.
+// TODO: extend Fleet's SearchField / InputFieldWithIcon to cover this
+// pattern — out of scope for the FleetsDropdown rework.
+@mixin menu-search-input {
+  width: 100%;
+  height: 36px;
+  box-sizing: border-box;
+  padding: 9.5px 12px 9.5px 36px;
+  line-height: $line-height;
+  font-family: "Inter", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)